restli-tools/src/main/java11/com/linkedin/restli/tools/idlgen/DocletDocsProvider.java

```diff
@@ -0,0 +1,267 @@
+/*
+   Copyright (c) 2012 LinkedIn Corp.
```

Comment:
as a matter of fact, since this is newly written, let's change the license and public doc

Reply:
Done.